test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12860,6 +12860,24 @@ def add_config(self, conf={}, hup=True):
 
         return self.apply_config(conf, hup)
 
+    def unset_mom_config(self, name, hup=True):
```

Comment:
Hi @hirenvadalia, I considered the name "delete_config" but that name really sounded like the function was deleting the whole configuration.  When I saw that the scheduler class had an "unset_sched_config" it seemed good to follow suit for consistency between classes.  
I do not want to rename add_config to set_mom_config since it is an existing PTL interface and it may already be used by others, and I do not want to affect those other tests/functions.

As for only allowing one configuration tunable to change at a time, it also seemed consistent to follow the same interface as the Scheduler class.  I also couldn't really see changing more than one mom configuration at a time...but I could be mistaken.  And even though it allows unsetting only one configuration tunable at a time, there is the option to not cause a restart each time...  I am interested to hear what others think.